src/rules/no-unused-modules.js

```diff
@@ -17,16 +17,20 @@ import includes from 'array-includes';
 // and has been moved to eslint/lib/cli-engine/file-enumerator in version 6
 let listFilesToProcess;
 try {
-  const FileEnumerator = require('eslint/lib/cli-engine/file-enumerator').FileEnumerator;
-  listFilesToProcess = function (src, extensions) {
-    const e = new FileEnumerator({
-      extensions: extensions,
-    });
-    return Array.from(e.iterateFiles(src), ({ filePath, ignored }) => ({
-      ignored,
-      filename: filePath,
-    }));
-  };
+  let { FileEnumerator } = require("eslint/use-at-your-own-risk");
+} catch {}
+
+try {
+    FileEnumerator = FileEnumerator || require('eslint/lib/cli-engine/file-enumerator').FileEnumerator;
```

Comment:
Rather than the conditional, let’s nest this inside the catch.

Reply:
And build [pyramid of doom](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming))?

Reply:
Yes, a thousand times over, since that's both correct and native language syntax.